File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        ans = [0] * (2 * n)
        for i in range(n):
            t = nums[i]
            ans[i] = t
            ans[i + n] = t
        return ans

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include <unordered_set>

using namespace std;

class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> uset;
        for(int i = 0; i < nums.size(); i++) {
            if(uset.count(nums[i]))
                return true;
            uset.insert(nums[i]);
        }
        return false;
    }
};